### PR TITLE
iso_c_binding: add statistics support.

### PR DESCRIPTION
**Pull request by fghoussen**
Should be the last PR !... :D Hope coverage will not break

### CODE REVIEW

**Comment by sylvestre** [comment]
We can regress the code coverage if we need to.

**Comment by fghoussen** [comment]
icb is "just" typing everything (function, suroutine, arguments, ...) : it's tedious + lot of declarations to (re-)do : so coverage may logically drop ! :D We'll see...

**Comment by sylvestre** [comment]
Well done, no regression in term of perf.
Shouldn't this change be mentioned in the changelog?

**Comment by fghoussen** [comment]
Sure ! I've added 2 lines in changelog and pushed force

File: README.md [hunk 1]
```diff
@@ -70,7 +70,9 @@ arpack-ng is replacing arpack almost everywhere.
 
    The following instructions explain how to make the ARPACK library.
 
-4. Unlike ARPACK, ARPACK-NG is providing autotools based build system.
+4. Unlike ARPACK, ARPACK-NG is providing autotools based build system
+   and iso_c_binding support (which enables to call fortran subroutines
```
Comment (sylvestre) [inline README.md]: maybe you could add a link to a page which explains what is iso_c_binding?

Comment (fghoussen) [reply README.md]: Woaw !... Not so easy. I try to get one.
I know a good one in french (in idris website - where I learnt everything from scratch), but, if I remember well this one is not translated in english !... I'll try to find something.

Comment (fghoussen) [reply README.md]: My understanding is that IDRIS lectures are french only. But I found:
http://fortranwiki.org/fortran/show/ISO_C_BINDING
http://fortranwiki.org/fortran/show/Generating+C+Interfaces

Do you buy one of these or both ?!

Comment (sylvestre) [reply README.md]: Yeah, clearly!

**Pull request merged**